src/Persist.js

```
@@ -1,45 +1,42 @@
-     * @param {Engine} engine - The engine class you wish to retrieve
-     * @return {Engine|null}
+     * Register a new connection.
+     * @param {string} name
+     * @param {StorageEngine} storage
+     * @param {Array<Model.constructor>} models
+     * @return {Connection}
      */
-    static getEngine(group, engine) {
-        return this._engine[group]?.[engine.name] ?? null;
+    static registerConnection(name, storage, models) {
+        this.#connections[name] = new Connection(storage, models);
```

Biome (1.9.4)

[error] Using this in a static context can be confusing.

this refers to the class.
Unsafe fix: Use the class name instead.


(lint/complexity/noThisInStatic)